src/dc_receive_imf.rs

```diff
@@ -2975,36 +2964,29 @@ mod tests {
             .unwrap();
         let chats = Chatlist::try_load(&t.ctx, 0, None, None).await.unwrap();
         assert_eq!(chats.len(), 1);
-        assert_eq!(chats.get_chat_id(0), deaddrop); // Test that the message is shown in the deaddrop
-
-        let msg = get_chat_msg(&t, deaddrop, 0, 1).await;
+        let chat_id = chats.get_chat_id(0);
```

Comment:
Could assert that the chat is a request here

Reply:
Done